pkg/environment/environment.go

```diff
@@ -0,0 +1,31 @@
+package environment
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetInt64 gets the env var as an int
+func GetInt64(varName string, defaultValue int64) int64 {
```

Comment:
we should start implementing these and others as a shared utils repo, as they're repeated on several repos

Reply:
Pocket-utils?

Reply:
yes but no, utils will suffice, obviously related to pocket but  naming would lead to understand they're directly related to pocket utils (like converting from public key to address) when this is more like code utils